src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -50,18 +102,18 @@
                 @{
                     var popOverContent = $"<b>{Model.ActiveConversations}</b> active revision threads.<br><b>{Model.UsageSampleConversations}</b> active usage sample threads.<br><b>{Model.TotalActiveConversations}</b> total active threads.<br>"
                     + $"<b>Current Revision:</b> <em>{@Model.Revision.DisplayName}</em>";
-                    @if (Model.DiffRevisionId != null)
+                                                @if (Model.DiffRevisionId != null)
                     {
                         popOverContent += $"<br><b>Current Diff:</b> <em>{@Model.DiffRevision?.DisplayName}</em>";
                     }
-                    <div class="btn-group btn-group-sm shadow-sm" aria-label="Page Information">
-                        <button type="button" class="btn btn-info" id="jump-to-first-comment">
-                            <i class="far fa-comment-alt"></i>
-                        </button>
-                        <button type="button" class="btn btn-info" data-placement="bottom" data-trigger="focus" data-toggle="popover" data-html="true" data-title="Page Info" data-content="@popOverContent">
-                            <span class="badge badge-light">@Model.ActiveConversations / @Model.TotalActiveConversations</span>
-                        </button>
-                    </div>
+                                                <div class="btn-group btn-group-sm shadow-sm" aria-label="Page Information">
+                                                    <button type="button" class="btn btn-info" id="jump-to-first-comment">
+                                                        <i class="far fa-comment-alt"></i>
+                                                    </button>
+                                                    <button type="button" class="btn btn-info" data-placement="bottom" data-trigger="focus" data-toggle="popover" data-html="true" data-title="Page Info" data-content="@popOverContent">
+                                                        <span class="badge badge-light">@Model.ActiveConversations / @Model.TotalActiveConversations</span>
+                                                    </button>
+                                                </div>

```

Comment:
This indentation appears strange, but in Visual Studio is automatically aligned to this

Reply:
I'm not behind my PC right now, but is this a spaces vs tabs issue?

Reply:
Potentially, but wouldn't it occur in more places if it was? I can investigate this possibility soon